app/src/main/shared/modules/jwtTokenAuth.ts

```diff
@@ -58,18 +58,17 @@ export const jwtTokenAuth = {
         const decoded = this.decodeToken(inflated);
         const permissions = decoded.permissions.split(",").filter((x: string) => x.length > 0);
         const modellingGroups = this.parseModellingGroups(decoded.roles);
-        return {
-            receivedBearerToken: true,
-            receivedCookies: false,
-            bearerToken: token,
-            isAccountActive: permissions.some((x: string) => x == "*/can-login"),
-            isModeller: modellingGroups.length > 0,
-            username: decoded.sub,
+
+        const result = loadAuthState(
+            decoded.sub, //username
```

Comment:
I think the commenting of args here suggests it'd be better to use an object as args for this function, i.e.
```
interface AuthStateOptions {
    username: string,
    receivedBearerToken: boolean,
    receivedCookies: boolean,
    bearerToken: string,
    permissions: string[],
    modellingGroups: string[]
}

export function loadAuthState(options: AuthStateOptions)
```

Advantages to doing it this way - can't make mistakes with param order, self-documenting, and better intellisense :smile: